### PR TITLE
Document HasTactic

### PR DESCRIPTION
**Pull request by AnonymousRandomPerson**

### CODE REVIEW

File: symbols/overlay29.yml [hunk 1]
```diff
@@ -3279,6 +3279,17 @@ overlay29:
...
+    - name: HasTactic
+      address:
+        EU: 0x230294C
+        NA: 0x2301F20
+        JP: 0x2303470
+      description: |-
+        Returns whether a monster is set to use a specific tactic.
+        
+        r0: Entity pointer
+        r1: Enum tactic_id
```
Comment (UsernameFodder) [inline symbols/overlay29.yml:3291]: This doesn't match the prototype?

**Review by UsernameFodder — CHANGES_REQUESTED** [review]

File: symbols/overlay29.yml (hunk 1 above)
Comment (AnonymousRandomPerson) [reply symbols/overlay29.yml:3291]: Oops, fixed.

File: symbols/overlay29.yml [hunk 2]
```diff
@@ -3279,6 +3279,17 @@ overlay29:
         
         r0: Entity pointer
         return: True if the monster can see invisible monsters.
+    - name: HasTactic
```
Comment (UsernameFodder) [inline symbols/overlay29.yml:3282]: I feel like "has" could potentially be interpreted as "is unlocked", which might be confusing. How about something like `IsTacticSet`?

Comment (AnonymousRandomPerson) [reply symbols/overlay29.yml]: Valid, I've renamed it.

**Review by UsernameFodder — APPROVED** [review]

**Pull request merged**